test/support/test_config.h

```diff
@@ -37,7 +37,7 @@
 // According to https://gcc.gnu.org/develop.html#timeline use last known _GLIBCXX_ to check the version of libstdc++
 #define _PSTL_TEST_LATEST_GLIBCXX 20240801

```

Comment:
@danhoeflinger This macro is no longer used by anyone and I suppose I should remove it.